AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -256,8 +257,47 @@ - (void)readWithPartition:(NSString *)partition
 - (void)createWithPartition:(NSString *)partition
                  documentId:(NSString *)documentId
                    document:(id<MSSerializableDocument>)document
-               writeOptions:(MSWriteOptions *)__unused writeOptions
+               writeOptions:(MSWriteOptions *)writeOptions
           completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler {
+  [self createOrReplaceWithPartition:partition
+                          documentId:documentId
+                            document:document
+                        writeOptions:writeOptions
+                   additionalHeaders:nil
+                   completionHandler:completionHandler];
+}
+
+- (void)deleteDocumentWithPartition:(NSString *)partition
+                         documentId:(NSString *)documentId
+                       writeOptions:(MSWriteOptions *)__unused writeOptions
+                  completionHandler:(MSDataSourceErrorCompletionHandler)completionHandler {
+  [self performOperationForPartition:partition
```

Comment:
Please add a task for cancellation on the azure board.

The Android client keeps track of pending calls and cancel them (and call completion handler with canceled error) when we disable the SDK (MSStorage.setEnabled:NO).

Reply:
@guperrot will do. which one is the Azure Board(Doug's Team)?

Reply:
No it should be on Yoann team like everything else related to storage.